sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSource.scala

```diff
@@ -769,7 +793,7 @@ object DataSource extends Logging {
       }
     }
 
-    allGlobPath
+    allPaths.seq
```

Comment:
Nit: can you call .toSeq? I'm working on Scala 2.13 support and I think this is deprecated

Reply:
https://github.com/apache/spark/pull/25899/commits/c52b3b08d9004c68b703bad73b27b348a63afc23